glpi/files/opt/startup.sh

```diff
@@ -76,6 +76,10 @@ do
 done
 
 # Run cron service.
+touch /var/log/cron-output.log
+touch /var/log/cron-errors.log
+tail -F /var/log/cron-output.log &
+tail -F /var/log/cron-errors.log &
```

Comment:
This permit to have this kind of output in docker logs:
![image](https://github.com/user-attachments/assets/c0aa9155-ad15-4aeb-84e6-61ce41e237d5)